packages/dx-react-grid/docs/reference/table-group-header.md

```diff
@@ -0,0 +1,60 @@
+# TableGroupHeader Plugin Reference
+
+A plugin that renders the nested header cells.
+
+## User Reference
+
+### Dependencies
+
+- [Table](table.md)
+- [TableHeaderRow](table-header-row.md)
+- [TableEditColumn](table-edit-column.md) [Optional]
```

Comment:
Why TableEditColumn, TableSelection, DragDropProvider, TableColumnResizing are optional? Seems like they are not dependencies at all.

Reply:
This plugin must place after all of this plugins. Because TableEditColumn, TableSelection, and TableColumnResizing add a cells/row to table head and TableGroupHeader is depended on this header rows and cells.